canonical-websites/jaas.ai
# Update /openstack content

## Done

- Remove the Autopilot section.
- Add the card js file so the cards load.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Visit /openstack.
- Check that there is no Autopilot section.
- Check that the cards load.

## Details

- Fixes: #106.
